### ISSUE TITLE
AstValueHelper.astFromValue() incorrectly handles Input types

### ISSUE THREAD

**Issue by anhldbk** [graphql-java/graphql-java#1719]
**Describe the bug**

`AstValueHelper.astFromValue()` currently fails when processing input types

**To Reproduce**

```java
import graphql.language.AstPrinter;
import graphql.language.AstValueHelper;
import graphql.language.Value;
import graphql.schema.GraphQLInputObjectField;
import graphql.schema.GraphQLInputObjectType;
import graphql.schema.GraphQLInputType;

import static graphql.Scalars.GraphQLString;

public class App {
  public static class Organization {
    public String name;

    static Organization of(String name) {
      Organization org = new Organization();
      org.name = name;
      return org;
    }
  }

  public static void test() {
    GraphQLInputType organizationInputType =
        GraphQLInputObjectType.newInputObject()
            .name("OrganizationInput")
            .field(
                GraphQLInputObjectField.newInputObjectField()
                    .type(GraphQLString)
                    .name("name")
                    .build())
            .build();

    Organization organization = Organization.of("Facebook");

    Value astValue = AstValueHelper.astFromValue(organization, organizationInputType);
    System.out.println(AstPrinter.printAst(astValue));
    // Currently, it prints
    //  { }
    // What I expect is: { name: "Facebook" }
  }

  public static void main(String[] args) {
    test();
  }
}
```
**Debug**

![Screen Shot 2019-12-21 at 15 09 57](https://user-images.githubusercontent.com/3270746/71305287-fa3c9e80-2403-11ea-85a8-d4c0a6f2714d.png)

`objToMap` seems NOT work as expected. Should I make a PR to address this issue?

**Comment by bbakerman** [comment]
AstValueHelper today only handles the following as documented

```

**Comment by andimarek** [comment]
this specific problem was solved with #1742. 

The more general challenge is tackled with #1745.
